man/amcrest-tui.1

```diff
@@ -0,0 +1,70 @@
+.TH man 1 "30 Jan, 2017" "1.0" "amcrest-tui man page"
+.SH NAME
+amcrest-cli \- A text user interface for amcrest camera
+.SH SYNOPSIS
+amcrest-tui
+.SH DESCRIPTION
+\fBamcrest-tui\fP is python3 only text user interface to visualize snapshots from Amcrest cameras.
+.br
+By default it reads the default ~/.config/amcrest.conf configuration file.
+.SH NOTES:
+NOTE1: As it uses framebuffer to visualize the snapshots, make sure your user has permission to manage /dev/fb0.
+.br
+NOTE2: If you use sudo command, make sure you put the amcrest.conf in /root/.config/amcrest.conf
+.P
```

Comment:
Let's add this note:

This option uses frrame buffer, which  requires your to run it directly from the Linux TTY console. Executing it from a  X terminal will not work.